main.rb

```diff
@@ -24,12 +24,8 @@ def runCommand(command)
     end
 end
 
-options = ""
-tuist_path = env_has_key("AC_TUIST_PATH")
+tuist_path = env_has_key("AC_TUIST_PATH") || abort("Missing AC_TUIST_PATH")
 tuist_commands = env_has_key("AC_TUIST_COMMANDS") || abort("Missing AC_TUIST_COMMANDS")
```

Comment:
Could you please move the "Missing KEY" error on lines 27 and 28 to the `env_has_key` function to avoid code duplication?